custom_components/tahoma/__init__.py

```diff
@@ -76,13 +76,25 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
     username = entry.data.get(CONF_USERNAME)
     password = entry.data.get(CONF_PASSWORD)
 
+    client = TahomaClient(username, password)
+
     try:
-        client = TahomaClient(username, password)
         await client.login()
+    except TooManyRequestsException:
+        _LOGGER.error("too_many_requests")
+        await client.close()
+        return False
+    except BadCredentialsException:
+        _LOGGER.error("invalid_auth")
```

Comment:
The log messages can be translated?

Reply:
No.. This was a placeholder for now, I think we need to pass the original exception messages.

Reply:
@tetienne @vlebourl what do you think? I am fine by having those short descriptive messages since this is part of the logging.

We should maybe reinit the config flow when we get a `BadCredentialsException`? Not sure how other components do that.

Reply:
In such case, I think the return False is enough. The user would have to remove and re add our integration to change its login/password. In can only occurs when it changes it itself.